src/Assert/Assert.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace Wikibase\DataModel\Services\Assert;
+
+use Wikimedia\Assert\Assert as WikimediaAssert;
+use Wikimedia\Assert\ParameterAssertionException;
+
+/**
+ * Provides functions to assure values meet certain preconditions relevant
+ * in Wikibase.
+ *
+ * @see Wikimedia\Assert\Assert
+ *
+ * @since 3.7
+ *
+ * @license GPL-2.0+
+ */
+class Assert {
```

Comment:
Yea... what other methods may we want to add here? If we had something to validate, say, site IDs, should it be in the same class? That kind of feels wrong...

Actually, the syntax is defined in the DataModel module, not in DataModelServices... so perhaps the assertions should be in DataModel too?